sensor/models.py

```diff
@@ -13,11 +13,30 @@
 
 from __future__ import unicode_literals
 
+from django.contrib.auth import get_user_model
 from django.db import models
 from django.urls import reverse
 
+User = get_user_model()
 
-class SensorType(models.Model):
+
+class SensorBaseModel(models.Model):
```

Comment:
I'd call this class `PermissionBase` or something like that and put it in the [`management` app models](https://github.com/ImperialCollegeLondon/paricia/blob/develop/management/models.py). From there, use it wherever is required.